README.md

```diff
@@ -1,3 +1,28 @@
-## About
+# About
 
-This is a (very) lightweight discord clone.
+Liscord is a (very) lightweight discord clone.
+
+# Requirements
+
+* Python 3.x
+
+# Installation
+
+* git clone https://github.com/raqibahmed/liscord
```

Comment:
Can we write what these steps do as well to make it more beginner friendly? Something like this can go above the command:

> First `cd` into a directory of your choice and clone the repository